tests/typecheck_examples.tests

```diff
@@ -717,25 +717,22 @@ args : --path=examples/sessions/mind
 
 Typecheck example file examples/mvu/mousetest.links
 examples/mvu/mousetest.links
-filemode : args
-args : --config=tests/typecheck_examples.tests.config
+filemode : true
 
 Typecheck example file examples/mvu/time.links
 examples/mvu/time.links
-filemode : args
-args : --config=tests/typecheck_examples.tests.config
+filemode : true
 
 Typecheck example file examples/mvu/pong.links
 examples/mvu/pong.links
-filemode : args
-args : --config=tests/typecheck_examples.tests.config
+filemode : true
+args : --config=foo
```

Comment:
WAT.

Do you have a config file named `foo`? In that case it seemingly hasn't been added to the revision control. I suspect it is a bogus name... but why?

Reply:
bollocks

Reply:
This was me testing for whether my fix actually worked. I'm going to pretend that I left it there just to be sure the CI broke.

:)